src/character.cpp

```diff
@@ -10906,32 +10905,29 @@ void Character::process_effects()
     }
 
     // Being stuck in tight spaces sucks. TODO: could be expanded to apply to non-vehicle conditions.
-    if( has_effect( effect_cramped_space ) && !in_vehicle ) {
-        remove_effect( effect_cramped_space );
-    }
-    // Check all of this here to ensure the player can't sit in a comfortable seat and then drop 50 liters of junk in their own lap.
-    if( in_vehicle ) {
+    if( has_effect( effect_cramped_space ) ) {
```

Comment:
The condition of the if block at line 10943 becomes always `false` after your change at this line, right?

Reply:
Oh, it looks like you're right. That shouldn't cause any problems but it should be removed.